rosdep/python.yaml

```
@@ -6592,6 +6592,15 @@ python3-pycryptodome:
       packages: [pycryptodome]
   rhel: ['python%{python3_pkgversion}-pycryptodomex']
   ubuntu: [python3-pycryptodome]
+python3-pydantic:
+  arch: [python-pydantic]
+  debian: null
+  fedora: [python-pydantic]
```

```suggestion
  fedora: [python3-pydantic]
```

https://fedora.pkgs.org/33/fedora-aarch64/python3-pydantic-1.6.1-1.fc33.noarch.rpm.html

`python-pydantic` key is obsolete AFAIS

---

thanks! i've updated it.